src/bindings/python/_flux/Makefile.am

```
@@ -173,15 +199,25 @@ lib-copy-vpath: ${fluxpyso_PYTHON} ${fluxpyso_LTLIBRARIES}
 		$(LN_S) .libs/$$LIB ./ $(STDERR_DEVNULL) || true; \
 	done
 
-all-local: lib-copy-vpath
+all-local: lib-copy-vpath ${fluxpyso_CODEGENJSON} ${fluxpyso_WRAPPERS}
+
+${fluxpyso_CODEGENJSON}:
+	$(AM_V_GEN)$(PYTHON) codegen.py dump --out $(srcdir)/codegen ${fluxpyso_CODEGENINPUT}
```

Just getting into trying this out, this line breaks with out of source builds (tries to use codegen.py from the build directory).  Think there's another one below the same.
```suggestion
	$(AM_V_GEN)$(PYTHON) $(top_builddir)/src/bindings/python/codegen.py dump --out $(srcdir)/codegen ${fluxpyso_CODEGENINPUT}
```